src/components/Coaching/CoachingRow/CoachingRow.tsx

```diff
@@ -0,0 +1,141 @@
+import { Box, styled, Typography } from '@material-ui/core';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { CoachFragment } from '../LoadCoachingList.generated';
+import { currencyFormat, percentageFormat } from 'src/lib/intlFormat';
+import StyledProgress from 'src/components/StyledProgress';
```

Comment:
Imports don't look right.

Reply:
The import should be good.
We have an absolute path setup in [tsconfig](https://github.com/CruGlobal/mpdx-react/blob/main/tsconfig.json) file.
I am pretty sure it provides more clean code.
https://nextjs.org/docs/advanced-features/module-path-aliases

Reply:
Ah interesting, I had issues locally when I had this come up. If it works for you and others then that's good with me! I may just have to figure out my local setup.